cdci_data_analysis/analysis/parameters.py

```diff
@@ -391,8 +400,11 @@ def check_value(val, units=None, name=None, par_format=None):
         pass
 
     def reprJSON(self):
-        return dict(name=self.name, units=self.units, value=self.value)
-    
+        # produces json-serialisable list
```

Comment:
ok, but could you name the function something like "reprJSONifiable"? I know it's long but it's less likely to cause mistakes.

Reply:
I used the existing function name. But ok, it's better to change

Reply:
Yeah, and approach like that probably caused dejsonifcation by trail in some dispatcher and oda_api places...